jgo/jgo.py

```diff
@@ -284,8 +284,23 @@ def _jgo_main(argv=sys.argv[1:], stdout=None, stderr=None):
 
     parser = jgo_parser()
 
-    completed_process = run(parser, argv=argv, stdout=stdout, stderr=stderr)
-    completed_process.check_returncode()
+    try:
+        completed_process = run(parser, argv=argv, stdout=stdout, stderr=stderr)
+        completed_process.check_returncode()
+
+    except HelpRequested:
+        pass
+        parser.print_help()
+
+    except NoEndpointProvided:
+        parser.print_usage()
+        _logger.error('No endpoint provided. Run `jgo --help\' for a detailed help message.')
```

Comment:
I don't think the "leading backtick, trailing single quote" quotation scheme is very common in technical output, is it? IMHO it would be better to enclose it in double quotes.

Reply:
I have seen it a lot in man pages, e.g. in `man zsh` (also [available for  download](http://zsh.sourceforge.net/Doc/zsh_info.tar.gz)). That being said I do not have a strong opinion on this and I do not have any preference either way, except that we should be consistent and change it in #49 as well, for example.

Reply:
Fair enough. And not worth spending any more time worrying about! Thanks.